static/src/js/components/Facets/Facets.js

```diff
@@ -138,6 +130,12 @@ const Facets = (props) => {
     autocompleteType: 'horizontal_data_resolution'
   }
 
+  const latencyResolution = {
+    ...cmrFacetDefaults,
+    title: 'Latency',
+    autocompleteType: 'latency'
```

Comment:
Should this be defined while the autocomplete is not working?

Reply:
It didn't cause any issues.. so figured we'd leave it.